migrations/20190327144239_cohort_table.js

```diff
@@ -0,0 +1,10 @@
+exports.up = function(knex) {
+    return knex.schema.createTable('cohorts', tbl => {
+		tbl.increments('id');
+        tbl.string('name', 128).notNullable();
```

Comment:
Also, putting each item on it own line and then indenting them the same makes your table more readable.